dev-docs/README.md

```diff
@@ -15,6 +15,7 @@ The following [project environment variables](https://circleci.com/docs/2.0/env-
 | `AWS_DEFAULT_REGION`           | Picked up by the AWS CLI              |
 | `AWS_ACCOUNT_ID`               | AWS account id                        |
 | `CIRCLECI_API_KEY`             | Used by the `queue` orb               |
+| `AWS_RESOURCE_NAME_PREFIX`     | 	Prefix that some of the required AWS resources are assumed to have in their names. The value should correspond to the `aws_resource_prefix` variable value in `terraform_setup/terraform.tfvars`.    |
```

Comment:
Actually, the project doesn't actually use this environment variable so this change can be omitted.